app/employee-app/backend/modules/database/types.bal

```diff
@@ -261,20 +266,53 @@ type DatabaseConfig record {|
     ConnectionPool connectionPool;
 |};
 
-# [Entity] Sub Team.
-public type SubTeam record {|
-    # Id of the sub team
+# Database config record.
+type HRISDatabaseConfig record {|
```

Comment:
Do we need to define a type? Can't we just create the object in client creation?

Reply:
You mean `HRISDatabaseConfig`?

Reply:
Just for my clarification. Can't we add options directly when the client is created?

Reply:
Resolved for now as discussed in the meeting.